auctionBidService/src/main/java/com/vmware/weathervane/auction/data/imageStore/NoBenchmarkInfoException.java~2e48a52... Removed the notion and code supporting database scales

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
There are a number of 'alternate version' files in this directory. (~HEAD, ~2e48a52...)
Are they needed?

Reply:
I have never seem that before.  I have removed the files.